install_config/configuring_authentication.adoc

```diff
@@ -419,17 +636,18 @@ password.
 
 // end::ldapblurb[]
 
-These are the steps taken:
 
-. Generate a search filter by combining the attribute and filter in the
+The following is the {product-title} process. {product-title}:
```

Comment:
this reads as "The following is the OpenShift Container Platform process. OpenShift Container Platform:" which doesn't make any sense